server/controllers/userController.js

```diff
@@ -116,15 +116,15 @@ class UserController {
 
   static async verifyEmailLink(req, res) {
     try {
-      const { token } = req.query;
-      const payload = auth.verifyMailToken(token);
-
-      if (!payload) return util.errorStatus(res, 400, 'Invalid Verification Link');
-
-      const { email } = payload;
-
+      const { token, id } = req.query;
```

Comment:
Before querying the DB, check the token and id value are defined else return Invalid link or something more descriptive

Reply:
Ok noted.